sphinx/source/docs/first_steps/first_steps_4.rst

```diff
@@ -58,8 +58,7 @@ Setting width and height
 
 To set the size of your plot, use the attributes
 :class:`~bokeh.models.plots.Plot.plot_height` and
-:class:`~bokeh.models.plots.Plot.plot_width` when calling the
-:func:`~bokeh.plotting.figure` function:
+:class:`~bokeh.models.plots.Plot.plot_width` when calling the |figure| function:
```

Comment:
I think this should `width` and `height` now

Reply:
Good point!